### ISSUE TITLE
less azure resources on multiple instances 

### ISSUE THREAD

**Issue by SecDWizar** [tfsaggregator/aggregator-cli#241]
# Feature Request

I've installed two instances with the same functions, one for production usage and another for development, 
There are double created resource for both (storage, ai, ...), can I use the same resources for both instances (except the function itself of course)

**Comment by giuliov** [comment]
No, [`install.instance`](https://tfsaggregator.github.io/docs/v3/commands/instance-commands/#installinstance) always create a new App Service. If you want to share more, simply use rules; add a suffix, say `-dev`, to a rule's clone and map it.
